### DISCUSSION TITLE
Welcome to FreedomWavePD Discussions!

### DISCUSSION THREAD

**Discussion by RChadwick7** [RChadwick7/FreedomWavePD#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ecomian** [comment]
Using your latest sofeware version 0.9.2 I get an error finger failure  fingers o to 7.  What does mean?

**Reply by RChadwick7** [reply]
What happens in ver0.9.2 on startup is that it tells the driver chips to adjust themselves to the LRA vibrators. If no vibrators are connected, or they are some other kind of actuator, this self-adjustment will fail and you'll see this error. What do you have connected to the controller?

**Comment by ecomian** [comment]
I have connected the gloves I made up (rough, but should be functional) and gone though the wiring doing a conductivity test on each finger from each vibrator lead to the board tracks where you haves labeled finger 0 to 8. Everything seems OK. 

I am using these vibrators   https://www.amazon.com/dp/B07JJFS99V?psc=1&ref=ppx_yo2ov_dt_b_product_details 

Will the way I have put the vibrators in the gloves be OK or do you think more pressure should be applied to the keep them pressing hard into the fingers?

Just found out that after it gets to the finger fail  8, it waits a couple of minutes and then displays minutes: 1,2,3,4 etc. Seems to

be going strait onto the treatment without any output to the gloves or menu with options.


Sunday: Just found out now that the gloves do make a faint noise and my son can feel a slight vibration that I cannot.  Seems that the chips are not putting enough power out to drive them. 
![IMG_20100101_120653](https://github.com/RChadwick7/FreedomWavePD/assets/161540414/400c7490-ded9-48f5-9ff2-dd8269eb738e)
![IMG_20100101_120631](https://github.com/RChadwick7/FreedomWavePD/assets/161540414/5f7f51b4-9b91-45f8-a3c5-61824499ee6e)
![IMG_20100101_120612](https://github.com/RChadwick7/FreedomWavePD/assets/161540414/7bcb120a-5ef5-4968-bdba-715bb034aed2)

**Comment by ecomian** [comment]
driver chips I used  https://www.ebay.com.au/itm/386666069677

**Comment by RChadwick7** [comment]
I see the mistake, and it's my fault. Sorry! Unfortunately, I was so focused on the control box I didn't verify the vibrators. The vibrators I had specified should be LRA, but are actually ERM. I updated the Github with the specific vibrators I used. You can also find them here: https://www.lcsc.com/product-detail/Vibration-Motor_LEADER-LD0832AA-0099F_C2682305.html. If you replace the vibrators you have with these, things should work fine. 
I should be coming out with a software update within the week that will enable menu functions, and you could then select ERM and they would work. Unfortunately, with ERM vibrators the vibrations come out from the sides of the disks. With LRA, they come out the top and bottom. If you wanted to re-use the vibrators you have, you could reposition them so the edge is touching the fingertip, and wait for the update to come out.

Also, once the menu update is working, I will be experimenting with, and hopefully releasing soon, the next version of the vibrators. They will use something other than LRA motors and have a pin that gently pushes into the skin similar to the ones developed by Dr. Peter Tass. I can't accurately predict when they will be released but my best guess is 1-4 months.

**Reply by ecomian** [reply]
Thanks, I will wait for the software update and new version. 

Is it possible pause the program if desired, or is that  not a good thing to do?

**Comment by RChadwick7** [comment]
The new update might be released as soon as tomorrow, but I've been very wrong about that in the past, so I'll say the new update hopefully will be released within the week. The new menu will Let you stop and restart treatment, but It doesn't have a pause. Not sure how that would affect treatment but I might add that in a later version.

**Reply by ecomian** [reply]
Do I have to have a mount for the Dayton DAEX-13-4SM ?    
Do you think they would work secured inside a fully enclosed glove? I am getting these gloves to try  https://www.ebay.com.au/itm/325875844401?var=514922808029). 
I am thinking of getting these  https://www.wagneronline.com.au/haptic-feedback-transducer-13mm-3w/haptic-feedback/exciters-tactile/speaker-drivers/audio-speakers-pa/daex-13-4sm-72443/994139/pd/ 

Can you point me to anyone else using your design? I want to know what protocols they are using and what results they are getting.?

Thanks you

**Comment by RChadwick7** [comment]
While I haven't personally used the DAEX-13, someone has made a housing for them. All the details are here: https://github.com/RChadwick7/FreedomWavePD/tree/main/Gloves/Parkgloveman%20DAEX%2013%20Mounting

Be aware that the latest released software probably won't work with these. The newest version is expected to be finished in a few days (I'm going away on business, I'll have time to finish it in the hotel) and will be adjustable for all the major parameters and should work with the Dayton exciters, as well as ERM. LRA open loop, LRA closed loop, change frequency.

As for who is using them, I have not received any feedback. My mother in law was who I used them on and she had perceptible improvements but nothing as drastic as the video shown on the Today Show. I'm hoping real actuators like the Dayton Audio will help them work better.

**Reply by ecomian** [reply]
I don't have the facilities to make a mount according to the details shown but thought I would just use the shrink wrap idea shared in the first part of this video using the DAEX-13.  https://www.youtube.com/watch?v=-9w1vbxou7E .
I think so far I have had some improvement in taste and smell and have been able to walk better but it is hard to tell because of so many other things going on, I will put them in a new set of motorbike gloves as soon as they arrive.

**Reply by RChadwick7** [reply]
The DAEX actuators are a bit like speakers and appear to be designed to be mounted in an enclosure. Using a soft housing may work but I suspect it will be delicate. The DAEX frame is plastic and easy to bend. Also, it may be important to hold the finger so that the actuator pin lightly touches the fingertip. Pushing hard into the actuator will likely suppress any perceptible vibration. I still need to research cost but I have seen a number of companies that will 3D print items for you. Now that the software is in a better state I'll be spending more time on actuators.

**Reply by RChadwick7** [reply]
Are you by chance located in the US?

**Comment by ecomian** [comment]
No I am in Australia
